tool/stats_lib.dart

```diff
@@ -118,10 +123,21 @@ CompareResult compareResult(
   bool verboseFail = false,
   bool verboseLooseMatch = false,
 }) {
+  var enableTagfilter = false;
+
+  for (final extension in testCase.extensions ?? []) {
```

Comment:
This can just use `testCase.extensions.contains`, yes?

Reply:
Yes, it is. I will change to `contains`. 

I was thinking of handling other extensions, but `testCase.extensions.contains` also works. maybe I was thinking there could be some extension combination which `contains` could not handle.